test/python/transpiler/test_cx_cancellation.py

```diff
@@ -1,199 +0,0 @@
-# This code is part of Qiskit.
-#
-# (C) Copyright IBM 2017, 2018.
-#
-# This code is licensed under the Apache License, Version 2.0. You may
-# obtain a copy of this license in the LICENSE.txt file in the root directory
-# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
-#
-# Any modifications or derivative works of this code must retain this
-# copyright notice, and modified files need to carry a notice indicating
-# that they have been altered from the originals.
-
-"""Tests for pass cancelling 2 consecutive CNOTs on the same qubits."""
-
-from qiskit import QuantumRegister, QuantumCircuit
-from qiskit.circuit import Clbit, Qubit
-from qiskit.transpiler import PassManager
-from qiskit.transpiler.passes import CXCancellation
-from test import QiskitTestCase  # pylint: disable=wrong-import-order
-
-
-class TestCXCancellation(QiskitTestCase):
```

Comment:
Would it make sense to move this test class to `test_inverse_cancellation.py` and modify the tests to use `InverseCancellation([CXGate])`?

Reply:
good point. Done in fe0e44c824ba3f3e1f133df9e10f0787da50216c . Thanks to that, I found this: https://github.com/Qiskit/qiskit/issues/13437